inc/utility.php

```diff
@@ -371,36 +376,29 @@ function cptui_newsletter_form() {
 	?>
 
 <div class="email-octopus-form-wrapper">
-    <p><strong>Get email updates from pluginize.com about Custom Post Type UI</strong></p>
-    <p class="email-octopus-success-message"></p>
-    <p class="email-octopus-error-message"></p>
-
-    <form method="post"
-      action="https://emailoctopus.com/lists/2039e001-4775-11ea-be00-06b4694bee2a/members/embedded/1.3/add"
-      class="email-octopus-form"
-      data-sitekey="6LdYsmsUAAAAAPXVTt-ovRsPIJ_IVhvYBBhGvRV6">
-        <div class="email-octopus-form-row">
-            <label for="field_0">Email address</label>
-            <input id="field_0" name="field_0" type="email" placeholder="email@domain.com" style="max-width:100%;">
-        </div>
-
-        <div class="email-octopus-form-row-hp" aria-hidden="true">
-            <!-- Do not remove this field, otherwise you risk bot sign-ups -->
-            <input type="text" name="hp2039e001-4775-11ea-be00-06b4694bee2a"
-               tabindex="-1"
-               autocomplete="nope">
-        </div>
-
-        <div class="email-octopus-form-row-subscribe">
-            <input type="hidden"
-               name="successRedirectUrl"
-               value="">
-            <button type="submit" class="button button-secondary">Subscribe</button>
-        </div>
-    </form>
+	<p><strong>Get email updates from pluginize.com about Custom Post Type UI</strong></p>
+	<p class="email-octopus-success-message"></p>
+	<p class="email-octopus-error-message"></p>
+
+	<form method="post" action="https://emailoctopus.com/lists/2039e001-4775-11ea-be00-06b4694bee2a/members/embedded/1.3/add" class="email-octopus-form" data-sitekey="6LdYsmsUAAAAAPXVTt-ovRsPIJ_IVhvYBBhGvRV6">
+		<div class="email-octopus-form-row">
+			<label for="field_0">Email address</label>
+			<input id="field_0" name="field_0" type="email" placeholder="email@domain.com" style="max-width:100%;">
+		</div>
+
+		<div class="email-octopus-form-row-hp" aria-hidden="true">
+			<!-- Do not remove this field, otherwise you risk bot sign-ups -->
+			<input type="text" name="hp2039e001-4775-11ea-be00-06b4694bee2a" tabindex="-1" autocomplete="nope">
+		</div>
+
+		<div class="email-octopus-form-row-subscribe">
+			<input type="hidden" name="successRedirectUrl" value="">
+			<button type="submit" class="button button-secondary">Subscribe</button>
+		</div>
+	</form>
```

Comment:
We should probably get the static content in this spot run through internationalization functions.